_data/bootloaders.json

```diff
@@ -29,10 +29,19 @@
             "family": "esp32s2",
             "bootloader_id": "adafruit_feather_esp32s2_tft",
         },
+        "adafruit_feather_rp2040": {
+            "family": "raspberrypi",
+        },
         "adafruit_funhouse": {
             "family": "esp32s2",
             "bootloader_id": "adafruit_funhouse_esp32s2",
```

Comment:
I don't think this originated with your PR, but while reviewing this I noticed that `bootloaders.json` contains trailing commas after every value. It seems these are not actually valid in JSON: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas#trailing_commas_in_json my IDE reports errors on almost every line due to these.
![bootloadresjsonerrors](https://user-images.githubusercontent.com/2406189/133290322-fa403d3d-c300-45f4-ad54-adc60b6d6a33.png)


I see that Jekyll is managing to parse the JSON correctly even with the trailing commas though, so it doesn't seem to be causing a problem in practice. I wonder if it's worth fixing so that we are matching the spec for JSON even though it's not currently broken. Perhaps this maybe better suited to a seperate issue though and not a part of this PR.

Reply:
I noticed those trailing commas after last-in-list values too. I'm happy to take them out in this PR, or wait.

Would people like to see the additional `mcufamily` additions in this PR, for near complete coverage of all 234 boards displayed, or go with what we have?

Reply:
A diff of `bootloaders.json` with trailing commas removed, plus `mcufamily` additions is at https://github.com/durapensa/circuitpython-org/commit/1115b64a19e858c9dc16ed5959ba7faa98f22dcc.

Generated from a python script, so there are a few alphabetical reorders & other json cleanups too.